dw-multi-value-field.js

```diff
@@ -366,7 +398,7 @@ export class DwMultiValueField extends DwFormElement(LitElement) {
     }
 
     //If validation is passsed then clear any errorMessage if was shown from previous validation,
-    this.errorMessage = '';
+    this.errorMessage = this.customValidator && !this.customValidator(value) ? this.customValidationMsg : ''; 
```

Comment:
custom validation jo false apse to ahiya sudhi avse j nhi., to ahiya kem avu lakhavani jarur padi?